docs/app/pages/components/components-sections/notifications/badge/badge.component.html

```diff
@@ -1,81 +1,79 @@
 <p>
-    <button class="btn btn-primary" [uxBadge]="9" [badgeSize]="'small'" [badgeTextColor]="'ok'">
+    <button class="btn btn-primary" [uxBadge]="9" badgeSize="small">
         Messages
     </button>
 </p>
 <p>
-    <button class="btn btn-primary" [uxBadge]="99" [badgeSize]="'small'" [badgeOverlap]="true"
-        [badgeBackgroundColor]="'vibrant1'">
+    <button class="btn btn-primary" [uxBadge]="99" badgeSize="small" badgeOverlap="true" badgeVerticalPosition="below"
+        badgeHorizontalPosition="before" badgeColor="vibrant1">
         Messages
     </button>
 </p>
 <p>
-    <button class="btn btn-primary" [uxBadge]="999" [badgeSize]="'small'" [badgeOverlap]="true"
-        [badgeBackgroundColor]="'warning'">
+    <button class="btn btn-primary" [uxBadge]="999" badgeSize="small" badgeOverlap="true" badgeVerticalPosition="bottom"
+        badgeHorizontalPosition="right" badgeColor="warning">
         Messages
     </button>
 </p>
 <p>
-    <button class="btn btn-primary" [uxBadge]="'NEW'" [badgeSize]="'small'">
+    <button class="btn btn-primary" [uxBadge]="'NEW'" badgeSize="small">
         Messages
     </button>
 </p>
 
 <hr />
 
 <p>
-    <button class="btn btn-primary" [uxBadge]="9" [badgeSize]="'medium'">
+    <button class="btn btn-primary" [uxBadge]="9" badgeSize="medium">
         Messages
     </button>
 </p>
 <p>
-    <button class="btn btn-primary" [uxBadge]="99" [badgeSize]="'medium'" [badgeOverlap]="true"
-        [badgeBackgroundColor]="'critical'">
+    <button class="btn btn-primary" [uxBadge]="99" badgeSize="medium" badgeOverlap="true" badgeColor="critical">
         Messages
     </button>
 </p>
 <p>
-    <button class="btn btn-primary" [uxBadge]="999" [badgeSize]="'medium'">
+    <button class="btn btn-primary" [uxBadge]="999" badgeSize="medium">
         Messages
     </button>
 </p>
 <p>
-    <button class="btn btn-primary" [uxBadge]="'NEW'" [badgeSize]="'medium'" [badgeOverlap]="true"
-        [badgeTextColor]="'critical'" [badgeAriaLabel]="'This is a custom aria label'">
+    <button class="btn btn-primary" [uxBadge]="'NEW'" badgeSize="medium" badgeOverlap="true"
+        badgeAriaLabel="This is a custom aria label">
         Messages
     </button>
 </p>
 
 <hr />
 
 <p>
-    <button class="btn btn-primary" [uxBadge]="9" [badgeSize]="'LARGE'" [badgeHidden]="hideButton">
+    <button class="btn btn-primary" [uxBadge]="9" badgeSize="LARGE" [badgeHidden]="hideButton">
         Messages
     </button>
     <button class="btn" (click)="hideButton = !hideButton">
         Hide/reveal badge
     </button>
 </p>
 <p>
-    <button class="btn btn-primary" [uxBadge]="99" [badgeSize]="'LarGE'">
+    <button class="btn btn-primary" [uxBadge]="99" badgeSize="LarGE">
```

Comment:
Hey was just testing this worked, these examples are not the final ones, just me testing that everything is working. I'll update this will final examples shortly.